.github/workflows/test.yml

```diff
@@ -49,14 +49,16 @@ jobs:
         run: go mod tidy -go=1.19 -compat=1.17 && git diff --exit-code
         if: ${{ matrix.go == '1.19' }}
       - name: Test
-        run: go test -count=1 ./...
```

Comment:
AFAIK we use `-count=1` to disable test result caching: https://stackoverflow.com/questions/73432741/why-does-count-1-ignores-caching-in-go-tests
But we could totally document this better.